src/App.jsx

```diff
@@ -4,6 +4,7 @@ import Account from "./Account.jsx";
 import Overlay from "./Overlay.jsx";
 import Login from "./Login.jsx";
 import React from "react";
+import NewOperator from "./NewOperator.jsx";
```

Comment:
Where are you using NewOperator?

Reply:
It was only a test and after that I forgot to delete the import...